code/modules/clothing/under/solgov.dm

```diff
@@ -36,6 +36,7 @@
 	desc = "A comfortable turtleneck and black utility trousers."
 	icon_state = "blackutility"
 	worn_state = "blackutility"
+	rolled_sleeves = 0
```

Comment:
Are there sprites for this?

Reply:
Base black one no. But if it cannot find them it simply states that it can't be rolled. There are sprites for all the others.

Reply:
It appears that the base black one does in fact have a sprite, so disregard my first comment.